elementary/monitor/alerts/alerts.py

```diff
@@ -22,9 +22,9 @@ def count(self) -> int:
         return len(self.alerts) + len(self.malformed_alerts)
 
     def get_all(self) -> List[Alert]:
```

Comment:
Isn't there a way to say that it's okay given that they both inherit from `Alert`?

Reply:
solved - I believe I did this before I added a bound to AlertType